__tests__/index.test.js

```diff
@@ -746,49 +824,62 @@ describe("Purchases", () => {
       return functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
     }
 
-    it("for functions that require the SDK to be configured if called before configuring", async () => {
-      NativeModules.RNPurchases.isConfigured.mockResolvedValue(false);
-
-      const allPropertyNames = Object.getOwnPropertyNames( Purchases );
-
-      // This functions should skip the test since they not required an instance of Purchases
-      const excludedFunctionNames = [
-        "configure",
-        "setSimulatesAskToBuyInSandbox",
-        "addCustomerInfoUpdateListener",
-        "removeCustomerInfoUpdateListener",
-        "addShouldPurchasePromoProductListener",
-        "removeShouldPurchasePromoProductListener",
-        "setAutomaticAppleSearchAdsAttributionCollection",
-        "addAttributionData",
-        "setDebugLogsEnabled",
-        "setLogLevel",
-        "setLogHandler",
-        "canMakePayments",
-        "UninitializedPurchasesError",
-        "UnsupportedPlatformError",
-        "throwIfNotConfigured",
-        "throwIfAndroidPlatform",
-        "convertIntToRefundRequestStatus",
-        "isConfigured",
-      ];
-      const expected = new Purchases.UninitializedPurchasesError();
-      for (let i = 0; i < allPropertyNames.length; i++) {
-        const property = Purchases[allPropertyNames[i]];
-        if (isFunction(property) && excludedFunctionNames.indexOf(allPropertyNames[i]) === -1) {
+    Purchases = require("../dist/index").default;
```

Comment:
I refactored this test so if it fails, the test name has the property that's failing